javascript/workflow/src/resources/TableBundle.ts

```diff
@@ -238,4 +245,12 @@ export class TableBundle extends Resource {
 			this.workflow.addInputs(this._inputs)
 		}
 	}
+
+	private encode(inputTable: Maybe<ColumnTable>) {
+		return this.codebook != null &&
+			this.codebook.fields.length > 0 &&
+			inputTable != null
+			? applyCodebook(inputTable, this.codebook, CodebookStrategy.MappingOnly)
```

Comment:
did the types + mapping not work?

Reply:
Nope, it blew up the integration tests

Reply:
do you have more info on what blew it up? what do we need to fix to make this work?

Reply:
In one case the table was empty, in another there was an internal processing error. Let me find a stack trace

Reply:
```
● nhefs-embedded › nhefs-embedded
@datashaper/workflow:test: 
@datashaper/workflow:test:     TypeError: value.replaceAll is not a function
@datashaper/workflow:test: 
@datashaper/workflow:test:       13 |   thousands = THOUSANDS_DEFAULT,
@datashaper/workflow:test:       14 | ): string {
@datashaper/workflow:test:     > 15 |   return value.replaceAll(thousands, '').replace(decimal, '.')
@datashaper/workflow:test:          |                ^
@datashaper/workflow:test:       16 | }
@datashaper/workflow:test:       17 |
@datashaper/workflow:test:       18 | export function getDate(value: string): Date {
@datashaper/workflow:test: 
@datashaper/workflow:test:       at replaceAll (../../.yarn/__virtual__/@datashaper-tables-virtual-cb2f1a5962/1/javascript/tables/src/util.ts:15:15)
@datashaper/workflow:test:       at formatNumberStr (../../.yarn/__virtual__/@datashaper-tables-virtual-cb2f1a5962/1/javascript/tables/src/parseTypes.ts:62:19)
```

Reply:
And with the other test case, the results came back empty: 
```
@datashaper/workflow:test:   ● nhefs-absolute-paths › nhefs-absolute-paths
@datashaper/workflow:test: 
@datashaper/workflow:test:     expect(received).toBeGreaterThan(expected)
@datashaper/workflow:test: 
@datashaper/workflow:test:     Matcher error: received value must be a number or bigint
@datashaper/workflow:test: 
@datashaper/workflow:test:     Received has value: undefined
@datashaper/workflow:test: 
@datashaper/workflow:test:       59 |                   expect(found).toBeDefined()
@datashaper/workflow:test:       60 |                   expect(found?.workflow?.length ?? 0).toEqual(table.workflowLength ?? 0)
@datashaper/workflow:test:     > 61 |                   expect(found?.output?.table?.numRows()).toBeGreaterThan(0)
@datashaper/workflow:test:          |                                                           ^
@datashaper/workflow:test:       62 |                   expect(found?.output?.table?.numCols()).toBeGreaterThan(0)
@datashaper/workflow:test:       63 |           }
@datashaper/workflow:test:       64 |           await checkPersisted(await datapackage.save(), expected)
@datashaper/workflow:test: 
```

Reply:
Hmm; the embedded spec doesn't even have a codebook, weird

Reply:
So the true error here is the first one I linked.  I'm not sure why it causes two fixtures to fail, but processing the codebok schema in that example is the root cause. 

I'll double-check the fixture's codebook schema tomorrow - this one could be pretty old